src/common/ChipCell/ChipCell.js

```diff
@@ -12,11 +12,19 @@ import ChipCellView from './ChipCellView'
 
 import { cutChips } from '../../utils/cutChips'
 import { sizeChips } from './SizeChips'
-import { panelActions } from '../../components/JobsPanel/panelReducer'
 
 import './chipCell.scss'
 
-const ChipCell = ({ className, dispatch, elements, isEditMode }) => {
+const ChipCell = ({
+  className,
+  elements,
+  addChip,
+  removeChip,
+  isEditMode,
+  isLabelsOnProjectOverview,
+  setIsLabelsOnProjectOverview,
```

Comment:
remove this prop from component and make it call in function where you need, this is a common component